README.md

```diff
@@ -2,4 +2,9 @@
 
 It holds sample code for a number of classic data structures implemented in Python.
 
+Current Data Structures in this repo:
+
+* Linked List
+* Stack
+
```

Comment:
tell me something about these structures here in your readme.  Don't just list them out like a menu